index.html

```diff
@@ -60,6 +58,87 @@ <h2 class="title text-primary">I’m a software developer</h2>
                 <img src="assets/images/Disabled.svg" alt="">
             </div>
         </section>
+        <section id="my-recent-work">
+            <h2 class="title 2">My Recent<br>Work<br></h2>
+                <div class="indicator"></div>
+                <div class="img-patholder">
+                    <div class="white-box">
+                        <div class="title 2">Multi-Post Stories <br>Gain+Glory</div>
+                        <div class="buttons">
+                            <button type="button">ruby on rails</button>
+                            <button type="button">Css</button>
+                            <button type="button">javascript</button>
+                            <button type="button">html</button>
+                        </div>
```

Comment:
- [ ] if you have a look at the Figma reference design you'll notice that the `font-weight` on each technology tag is actually greater than what you currently have. please use the correct `font-size` in this section

| Figma Design | Your Design |
-|-
|![image](https://user-images.githubusercontent.com/18730106/153636665-5f691722-b965-4780-bb9b-0b46eee7c5ce.png)|![image](https://user-images.githubusercontent.com/18730106/153636694-85f59697-0c6b-4165-91c6-9a7640cd6a44.png)|